scripts/changelog.sh

```diff
@@ -3,8 +3,6 @@
 set -o pipefail
 
 tfile=$(mktemp /tmp/doctl-CHANGELOG-XXXXXX)
-github-release-notes -org digitalocean -repo doctl -since-latest-release -include-author >"$tfile"
-
-GO111MODULE=on go mod tidy
```

Comment:
You'll need the `go mod tidy` to prevent ghcg from ending up in `go.mod`

Reply:
This is done, not sure why GitHub isn't picking up the change

Reply:
I see it :)